core/layers/svc-core-proxy/extension/clients/elasticsearch_client.go

```
@@ -0,0 +1,106 @@
+
+	response, err := client.Search(client.Search.WithIndex(ag), client.Search.WithBody(&buffer))
+	if err != nil {
+		return nil, fmt.Errorf("error getting response from opensearch client for asset id [%s] %w", assetId, err)
+	}
+	defer response.Body.Close()
+
+	if response.StatusCode != 200 {
+		return nil, fmt.Errorf("error while fetching asset details from opensearch client for asset id [%s]", assetId)
+	}
+	var result map[string]interface{}
+	json.NewDecoder(response.Body).Decode(&result)
```

golangci-lint (1.62.2)

Error return value of `(*encoding/json.Decoder).Decode` is not checked

(errcheck)